app/code/Magento/Elasticsearch6/Model/Client/Elasticsearch.php

```diff
@@ -113,6 +113,11 @@ private function buildConfig($options = [])
         if (!empty($options['port'])) {
             $host .= ':' . $options['port'];
         }
+        
+        if (isset($options['enableAuth']) && ($options['enableAuth'] == 0)) {
```

Comment:
I would favor code consistency, thus my suggestion for this method is that of replacing all the occurrences of `empty()`  with `isset()` (which I personally favor) or using empty()` as well. What do you think?

Reply:
I don't know what the rest of the code does when changing everything to isset. I know that 0 is consider empty this is why I used isset in the fix.

I only fixed the problem I didn't try to refactor the method or intended to refactor it. This is however a real issue, which in my opinion should be fixed asap because it is broken. How the fix is refactored in a later stage of development is not my concern.